src/main/java/org/ibp/api/rest/labelprinting/ObservationDatasetLabelPrinting.java

```diff
@@ -54,13 +55,13 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Set;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 @Component
 @Transactional
 public class ObservationDatasetLabelPrinting extends LabelPrintingStrategy {
 
+	private static final String UNDERSCORE = "_";
```

Comment:
@cuenyad I can see this static defined in many places, it can be defines in LabelPrintingFieldUtil and reuse it from there

Reply:
done!